core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -56,7 +60,7 @@ <h1 class="stat-value"><[dashboardStats.total_open_feedback]></h1>
         </li>
         {% if not can_create_collections %}
           <!-- A layout fix when the 'My Collections' tab is absent -->
-          <li ng-if="collectionsList.length == 0"></li>
+          <li ng-if="collectionsList.length === 0 && subscribersList.length === 0"></li>
```

Comment:
One thought: do you need to show the Subscribers tab if the length of the list of subscribers is 0?